src/ipc.rs

```diff
@@ -62,12 +62,12 @@ pub fn bytes_channel() -> Result<(IpcBytesSender, IpcBytesReceiver),Error> {
 }
 
 #[derive(Debug)]
-pub struct IpcReceiver<T> where T: Deserialize + Serialize {
+pub struct IpcReceiver<T> where T: for<'de> Deserialize<'de> + Serialize {
```

Comment:
I'd check with @dtolnay, but I'm pretty sure that since you can't taking advantage of zero-copy parsing, you could replace all of these lifetime bounds with `Deserialize<'static>`.